lib/feeder_ex.ex

```diff
@@ -22,6 +22,11 @@ defmodule FeederEx do
     :feeder.stream feed_bin, opts
   end
 
+  def parse!(feed_bin) do
+    {:ok, feed, _} = :feeder.stream feed_bin, opts
```

Comment:
i think `{:ok, feed, _} = parse(feed_bin)` would be even nicer here. we could avoid some duplication then. sorry for giving you the wrong idea before.